lstmcpipe/stages/mc_r0_to_dl1.py

```diff
@@ -320,75 +244,50 @@ def main(input_dir, config_file=None, train_test_ratio=0.5, random_seed=42, n_r0
 
-            # recover or not the jobid depending of the workflow mode
-            if not flag_full_workflow:  # Run interactively
-
-                cmd = f'sbatch -p short -e {jobe} -o {jobo} {base_cmd} {output_dir} {os.path.join(dir_lists, file)}'
-                # print(cmd)
-                os.system(cmd)
-
-            else:  # flag_full_workflow == True !
-                job_name = {'electron': f'e_{jobtype_id}_r0dl1',
-                            'gamma': f'g_{jobtype_id}_r0dl1',
-                            'gamma-diffuse': f'gd_{jobtype_id}_r0dl1',
-                            'proton': f'p_{jobtype_id}_r0dl1',
-                            'gamma_off0.0deg': f'g0.0_{jobtype_id}_r0dl1',
-                            'gamma_off0.4deg': f'g0.4_{jobtype_id}_r0dl1'
-                            }
-
-                if particle == 'proton':
-                    queue = 'long'
-                else:
-                    queue = 'short'
-
-                cmd = f'sbatch --parsable -p {queue} -J {job_name[particle]} ' \
-                      f'-e {jobe} -o {jobo} {base_cmd} {output_dir} {os.path.join(dir_lists, file)}'
-
-                jobid = os.popen(cmd).read().strip('\n')
-                jobids_r0_dl1.append(jobid)
-
-                # Fill the dictionaries if IN workflow mode
-                jobid2log[jobid] = {}
-                jobid2log[jobid]['particle'] = particle
-                jobid2log[jobid]['set_type'] = set_type
-                jobid2log[jobid]['jobe_path'] = jobe
-                jobid2log[jobid]['jobo_path'] = jobo
-                jobid2log[jobid]['sbatch_command'] = cmd
-
-                # print(f'\t\t{cmd}')
-                # print(f'\t\tSubmitted batch job {jobid}')
-                save_job_ids.append(jobid)
+            job_name = {'electron': f'e_{jobtype_id}_r0dl1',
+                        'gamma': f'g_{jobtype_id}_r0dl1',
+                        'gamma-diffuse': f'gd_{jobtype_id}_r0dl1',
+                        'proton': f'p_{jobtype_id}_r0dl1',
+                        'gamma_off0.0deg': f'g0.0_{jobtype_id}_r0dl1',
+                        'gamma_off0.4deg': f'g0.4_{jobtype_id}_r0dl1'
+                        }
+
+            if particle == 'proton':
+                queue = 'long'
+            else:
+                queue = 'short'
+
+            cmd = f'sbatch --parsable -p {queue} -J {job_name[particle]} ' \
+                  f'-e {jobe} -o {jobo} {base_cmd} {output_dir} {os.path.join(dir_lists, file)}'
+
+            jobid = os.popen(cmd).read().strip('\n')
+            jobids_r0_dl1.append(jobid)
+
+            # Fill the dictionaries if IN workflow mode
+            jobid2log[jobid] = {}
+            jobid2log[jobid]['particle'] = particle
+            jobid2log[jobid]['set_type'] = set_type
+            jobid2log[jobid]['jobe_path'] = jobe
+            jobid2log[jobid]['jobo_path'] = jobo
+            jobid2log[jobid]['sbatch_command'] = cmd
+
+            # print(f'\t\t{cmd}')
```

Comment:
We should really just move to logging and mark those as debug :D
But that can be done in another PR

Reply:
Better another PR, yes ;)